molecule/active-ubuntu2004/molecule.yml

```diff
@@ -16,6 +18,10 @@ platforms:
       - /sys/fs/cgroup
 provisioner:
   name: ansible
+  config_options:
```

Comment:
Can you add a comment explaining why this is required so that we can come back to it later?

Reply:
I will remove, this was added in attempt to fix a problem which somehow `went away by itself`.

Reply:
fixed in https://github.com/ome/ansible-role-omero-web/pull/45/commits/b8d0468564388e0ad9c75fd810dc4f774fa6c793